### DISCUSSION TITLE
Frequency Track

### DISCUSSION THREAD

**Discussion by jallmer** [GMOD/jbrowse-components#1572] [Q&A]
Hi, 
I want to visualize a number per base just like a coverage.
However, I don't have underlying sequence data (stability calculations).
It would be straightforward for me to create a bedgraph, but that doesn't seem to be supported in jbrowse 2.

So my problem is I cannot go via SAM/BAM since there are no reads or alignments.
Any suggestions on how to achieve the visualization?

Thanks

**Comment by cmdcolin** [comment]
I mentioned on the chatroom too but I think maybe consider using bedGraphToBigWig from UCSC (http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/)


This will convert to bigwig, which we then support. We could make a data adapter for bedgraph filetype natively, but it would likely not perform as well as simply converting to bigwig would

**Comment by cmdcolin** [comment]
Also, just while we are discussing it, do you have any details on the type of data? I know for something like perhaps GWAS then it is helpful to have the SNP names and their scores

In that case, things like this existed in jbrowse 1 https://github.com/elsiklab/gwasviewer but would need some work to port the similar idea to jbrowse 2!

**Reply by jallmer** [reply]
This might be a particular use-case.
I used RNAFold to fold a genome using different-sized portions (80,160,250,500,...). 
For each position, I counted in how many folds there was a bond and used that as an accessibility value.
Now at each position, I have a value ranging from 0 to 9.
Translating that into a bedgraph is trivial, and scaling (if necessary, pending visualization) can be done.
Maybe some other approaches than the one mentioned by you earlier are possible, but I'll give the conversion tool a shot first.
